tests/integration/api/test_api.py

```diff
@@ -553,6 +554,17 @@ def _compare_screenboard(board1, board2):
 
         share_res = dog.Screenboard.share(get_res['id'])
         assert share_res['board_id'] == get_res['id']
+        public_url = share_res['public_url'].replace('8080', '5000')
```

Comment:
I am usually testing against staging or a prod demo account, so I never experienced this inconvenience before.
Still, I don't think we should commit this extra line: it's too much conditioned by your environment thus it could easily break in a different scenario (i.e. testing from your VM or not).

Maybe we can make this more flexible using an environment variable like we do to pass the Datadog URL ?